apps/blockchain/test/blockchain/state_test.exs

```diff
@@ -1927,7 +1927,7 @@ defmodule Blockchain.StateTest do
     # "stSystemOperationsTest/suicideCoinbase.json",
     "stSystemOperationsTest/suicideNotExistingAccount.json",
     "stSystemOperationsTest/suicideOrigin.json",
-    # "stSystemOperationsTest/suicideSendEtherPostDeath.json",
+    "stSystemOperationsTest/suicideSendEtherPostDeath.json",
```

Comment:
That's strange. Shouldn't this have already been uncommented? Did we forget to do that @masonforest?

Reply:
yes, it looks like so